docs/capturing-arguments.md

```diff
@@ -0,0 +1,107 @@
+# Capturing arguments
+
+There are a large number of [argument constraints](argument-constraints.md)
+that FakeItEasy provides to restrict which calls are being configured or to
+verify that particular calls were made, and custom ones can be added, but
+sometimes they aren't sufficient and it is desirable to capture
+arguments supplied in calls to fake objects. Here's how to do that.
+
+## Simple capture
+
+Take this contrived example:
+
+```csharp title='"production code"'
+--8<--
+recipes/FakeItEasy.Recipes.CSharp/CapturingArguments.cs:IListLogger
+
+recipes/FakeItEasy.Recipes.CSharp/CapturingArguments.cs:Popper
+--8<--
+```
+
+While there are other ways to ensure that a particular list is passed
+to a method, we'll pretend there aren't and capture the second argument
+to `log` in a `CapturedArgument`, then verify it.

```

Comment:
This is a bit confusing, I had to read it twice to understand what it was about.
I think it's because it's written with the idea of "working around limitations of argument constraints" in mind, which isn't the use case _I_ have in mind when I think about argument capture...